tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -589,6 +629,7 @@ void SPIRVEmitter::doFunctionDecl(const FunctionDecl *decl) {
     // calls. We have already assigned <result-id>s for it when translating
     // its call site. Query it here.
     funcId = declIdMapper.getDeclResultId(decl);
+    assert(funcId);
```

Comment:
the above suggestion will eliminate the need for this assert.

Reply:
Done